lib/search_solr_tools/config/environments.rb

```diff
@@ -5,7 +5,7 @@
 module SearchSolrTools
   # configuration to work with solr locally, or on integration/qa/staging/prod
   module SolrEnvironments
-    YAML_ENVS = YAML.load_file(File.expand_path('environments.yaml', __dir__))
+    YAML_ENVS = YAML.load_file(File.expand_path('environments.yaml', __dir__), aliases: true)
 
     def self.[](env = :development)
       YAML_ENVS[:common].merge(YAML_ENVS[env.to_sym])
```

Comment:
Do you know of a reason why `common` can't be integrated into each environment in the YAML file itself (using `<<: *common`? The `dev`/`development` environment might need some refactoring, but the others could be handled in the config file itself, I think.

Reply:
I'm not sure why the environment.yaml file was written this way with the rb file merging them.

I'll try updating the yaml file, and then the merge on line 11 wouldn't need to be there.